stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -264,13 +264,13 @@ spec:
       {{- if .Values.schedulerName }}
       schedulerName: "{{ .Values.schedulerName }}"
       {{- end }}
+
       {{- if .Values.affinity }}
       affinity:
       {{- with .Values.affinity }}
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- else }}
-      {{- if eq .Values.podAntiAffinity "hard" }}
+      {{- else if eq .Values.podAntiAffinity "hard" }}
```

Comment:
I don't see how changing an `else` with an inner `if` into an `else if` would change the boolean logic at all in this code. In either scenario, if you're supplying a `.Values.affinitity` it would take precedence and the podAntiAffinity would not be used. And if you don't supply a `.Values.affinity` the podAntiAffinity would be used.

Reply:
I tested this again and it worked as expected, sorry for the bother.